python/lsst/ts/watcher/rules/mt_hexapod_high_current.py

```diff
@@ -0,0 +1,121 @@
+# This file is part of ts_watcher.
+#
+# Developed for Vera C. Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["MTHexapodHighCurrent"]
+
+
+import logging
+import types
+import typing
+
+from lsst.ts import salobj, watcher
+from lsst.ts.xml.enums.MTHexapod import SalIndex
+from lsst.ts.xml.enums.Watcher import AlarmSeverity
+
+from ..base_rule import AlarmSeverityReasonType, NoneNoReason
+from ..remote_info import RemoteInfo
+
+
+class MTHexapodHighCurrent(watcher.BaseRule):
+    """Monitor the high motor current event of the MT hexapod.
+
+    Parameters
+    ----------
+    config : `types.SimpleNamespace`
+        Rule configuration, as validated by the schema.
+    log : `logging.Logger` or None, optional
+        Parent logger. (the default is None)
+    """
+
+    def __init__(
+        self, config: types.SimpleNamespace, log: logging.Logger | None = None
+    ):
+
+        remote_name = "MTHexapod"
+        remote_info_camera = RemoteInfo(
```

Comment:
I am not sure compounding the 2 hexapods into the same rule is a good idea. It means we cannot independently control muting/acknowledging etc of the alarms.  I think we probably want an alarm that takes the CSC index (or name->index mapping) as a configuration.